account/keys.go

```diff
@@ -0,0 +1,61 @@
+package account
+
+import "github.com/tendermint/go-crypto"
+
+// This allows us to control serialisation
+
+type PublicKey struct {
+	crypto.PubKey `json:"unwrap"`
+}
+
+func PublicKeyFromPubKey(pubKey crypto.PubKey) PublicKey {
+	return PublicKey{PubKey: pubKey}
+}
+
+func PrivateKeyFromPrivKey(privKey crypto.PrivKey) PrivateKey {
```

Comment:
`PublicKeyFromPubKey` and `PrivateKeyFromPrivKey` are weird names. It's not obvious what they do without context. Suggest maybe `P*KeyFromCryptoP*Key` or maybe just omitting these functions since they add an extra layer of indirection without adding much.

Reply:
I agree on the names. I sort of intended the indirection though. I'd like a single place to hook any changes in dependency. Currently it's a simple wrapping but it could be a true mapping. I'm going to hold my nose and favour the full `PrivateKeyFromGoCryptoPrivKey`